supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -788,13 +788,13 @@ image repository contains one or more tagged images.
 [discrete]
 [[control-node]]
 ==== image:images/yes.png[yes] control node (noun)
-*Description*: Any machine with Ansible installed. You can run commands and playbooks, invoking /usr/bin/ansible or /usr/bin/ansible-playbook, from any _control node_. You can use any computer that has Python installed on it as a control node - laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
+*Description*: Any machine with Ansible installed. You can run commands and playbooks, invoking `/usr/bin/ansible` or `/usr/bin/ansible-playbook`, from any _control node_. You can use any computer that has Python installed on it as a control node: laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
```

Comment:
I think the last sentence, _You can have multiple control nodes_, should be moved elsewhere as it feels out of place with the Windows sentence that comes before it. The can > cannot > can paragraph construction is a bit off. Perhaps finish with the Windows sentence?